packages/@aws-cdk/aws-codepipeline-actions/package.json

```diff
@@ -151,6 +151,7 @@
       "props-default-doc:@aws-cdk/aws-codepipeline-actions.ManualApprovalActionProps.additionalInformation",
       "props-default-doc:@aws-cdk/aws-codepipeline-actions.ManualApprovalActionProps.notificationTopic",
       "props-default-doc:@aws-cdk/aws-codepipeline-actions.ManualApprovalActionProps.notifyEmails",
+      "props-default-doc:@aws-cdk/aws-codepipeline-actions.ManualApprovalActionProps.externalEntityLink",
```

Comment:
To make this warning go away, you need to provide documentation for the default value of the property, like this:

```ts
  /**
   * URL you want to provide to the reviewer as part of the approval request.
   * The URL must begin with 'http://' or 'https://'.
   *
   * @default the approval request will not have an external link
   */
  readonly externalEntityLink?: string;
```

Reply:
thanks